src/platform/forms-system/src/js/utilities/review/index.js

```diff
@@ -35,29 +28,36 @@ const findTargets = (error, { getLabel = true } = {}) => {
     ].join(','),
   );
   if (error.pageKey === error.chapterKey) {
-    return { scroll: el, focus: el };
+    return { scroll: el };
   }
-  // Find all visible elements within the form
-  const form = el?.closest('.form-review-panel-page')?.querySelectorAll('form');
-  const formElements = form
-    ? getFocusableElements(form[error.index] || form[0])
-    : [];
-  // narrow down search to a matching id (if possible)
-  const firstElement =
-    formElements.filter(elm => elm.id.includes(`_${error.name}`))[0] ||
-    // ID may not match the name in an array block (e.g. 526 servicePeriods)
-    formElements[0];
   return {
     scroll: el,
-    focus:
-      (getLabel
-        ? firstElement
-            ?.closest('.schemaform-field-container, .schemaform-field-template')
-            ?.querySelector('legend, label')
-        : firstElement) || el,
   };
 };
 
+/**
+ * Open the accordion for the specified chapter and click on the "Edit" button
+ * to go into edit mode. This will also result in the focusing of the first
+ * interacrtive element in the content.
+ * @param {String} chapterKey - a chapter key
+ */
+export const openAndEditChapter = chapterKey => {
+  const accordionItem = document.querySelector(
+    `va-accordion-item[data-chapter="${chapterKey}"`,
+  );
+
+  const accordionItemButton = accordionItem.shadowRoot.querySelector(
+    "button[aria-controls='content']",
+  );
+
+  const editButton = accordionItem.querySelector('va-button');
```

Comment:
Can we pass in the pageKey to this function? This would only target the first page inside a chapter, which may not be the page that needs to be edited.

Reply:
Great catch. I was hoping to get all the tests green before pinging you for a followup, but I see you're just on top of things. :)


https://github.com/department-of-veterans-affairs/vets-website/assets/2482403/53d70dfc-e977-4e3d-b397-30c9a50cf14b